Program.fs

```diff
@@ -71,9 +72,9 @@ let needsNodeModules (packageJson: string) =
     let siblings = [ yield! IO.Directory.GetFiles parentDir.FullName; yield! IO.Directory.GetDirectories parentDir.FullName ]
     let nodeModulesExists = siblings |> List.exists (fun file -> file.EndsWith "node_modules")
     let yarnLockExists = siblings |> List.exists (fun file -> file.EndsWith "yarn.lock")
-    if nodeModulesExists then None
-    elif yarnLockExists then Some "yarn install"
-    else Some "npm install"
+    if nodeModulesExists then false
+    elif yarnLockExists then true
```

Comment:
This can be simplified even more using `not nodeModulesExists` You can also remove the check for `yarn.lock`